### PR TITLE
Routine updates for JDK11 build compatibility

### PR DESCRIPTION
**Pull request by Vlatombe**
* Update parent POM
* Use BOM

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by bitwiseman — APPROVED** [review]
I'd prefer jenkins 2.249.x LTS if possible.

**Review by bitwiseman — APPROVED** [review]
I'd prefer jenkins 2.249.x LTS if possible.

**Review by alecharp — APPROVED** [review]

**Comment by mikecirioli** [comment]
gentle nudge @Wadeck && @daniel-beck

**Comment by Wadeck** [comment]
@mikecirioli sorry, no time at the moment, preparing next security release. Perhaps after Jan 12 we will have more time.

Is it blocking you?

**Comment by mikecirioli** [comment]
its not urgently blocking @Wadeck - if you can find a few minutes after jan 12th gaia team would appreciate it!

**Comment by mikecirioli** [comment]
@Wadeck gentle nudge 😉

File: pom.xml [hunk 1]
```diff
@@ -72,23 +72,32 @@
...
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
-      <version>2.2.0</version>
```
Comment (daniel-beck) [inline pom.xml:79]: Dependency is increased to 2.6.1 through the BOM.

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**